src/main/jbake/content/developer/jms/advanced.adoc

```diff
@@ -0,0 +1,109 @@
+= JMS Advanced
+:jbake-date: 2016-03-17
+:jbake-type: page
+:jbake-status: published
+:jbake-tomeepdf:
+
+
+=== Configuring the Container tomee.xml
+
+===== Queue
+
+[source,xml]
+----
+<Resource id="Foo" type="javax.jms.Queue">
+    destination = Specifies the name of the queue
+</Resource>
+----
+
+===== Topic
+
+[source,xml]
+----
+<Resource id="Foo" type="javax.jms.Topic">
+    destination = Specifies the name of the topic
+</Resource>
+----
+
+
+===== ConnectionFactory
+
+[source,xml]
+----
+<Resource id="Foo" type="javax.jms.ConnectionFactory">
+    ResourceAdapter = Default value is Default JMS Resource Adapter.
+    TransactionSupport = Specifies if the connection is enrolled in global transaction, allowed values: xa, local or none. Default value is xa.
+    PoolMaxSize = Maximum number of physical connection to the ActiveMQ broker. Default value is 10.
+    PoolMinSize  = Minimum number of physical connection to the ActiveMQ broker. Default value is 0.
+    ConnectionMaxWaitMilliseconds = Maximum amount of time to wait for a connection. Default value is 5000.
+    ConnectionMaxIdleMinutes = Maximum amount of time a connection can be idle before being reclaimed. Default value is 15.
+</Resource>
+----
+
+===== MESSAGE
+
+[source,xml]
+----
+<Container id="Foo" type="MESSAGE">
+    ResourceAdapter = The resource adapter delivers messages to the container. Default value is Default JMS Resource Adapter.
+    MessageListenerInterface = Specifies the message listener interface handled by this container. Default value is javax.jms.MessageListener.
+    ActivationSpecClass = Specifies the activation spec class. Default value is org.apache.activemq.ra.ActiveMQActivationSpec.
+    InstanceLimit = Specifies the maximum number of bean instances that are allowed to exist for each MDB deployment. Default value is 10.
+</Container>
+----
+
+
+=== Activation Message
+
+A JMS activation specification is associated with one or more message-driven beans and provides the configuration necessary for them to receive messages.
+
+
+[source,java]
+----
+containerId.activation.atrribute = value
+----
+
+So, the attribute will send through all the message-drive bean who has the container id equals to *containerId*.
+
+
+==== Global Configuration
+
+To set a global configuration which will be used for all container configured we can you can use `mdb` instead of `containerId`
+
+[source,java]
+----
+mdb.activation.atrribute = value
+----
+
+==== Ignore fields:
+
+To ignore any field from Activation message, override the *failOnUnknownActivationSpec* to _false_ at MdbContainer class.
```

Comment:
A `service-jar.xml` file isn't actually needed. Just adding a container to `tomee.xml` with `FailOnUnknownActivationSpec = false` is sufficient. 

Covering `service-jar.xml` is useful topic, but I'd cover that in its own right, as opposed it being a small part of JMS.